OKR.md

```diff
@@ -2,6 +2,32 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q3
+
+**The Connectivity Magic story is figured out and solved**
+
+- A js-ipfs Daemon will always find other IPFS Daemons (js & go) in the LAN 
+- A js-ipfs node (Browser & Desktop) always finds files that are on the Gateways
+- The rendezvous protocol is a sound solution for DApps (e.g. PeerPad & others)
+- The DHT is enabled by default in js-ipfs
+- Handles correctly a scenario where the number of Connections grows (through Conn Management Strategy)
+
+**Daemon is as reliable as it can be**
+
+- A Daemon runs for 10 days without a crash
+- One or more js-ipfs daemons are part of the Bootsrapper nodes 
+- js-ipfs can handle 1TB of data, both Node.js and Browser - @hugomrdias
+- npm on IPFS over js-ipfs is a sound way to install deps and gets used for CI - @achingbrain
+
+**New user/contributor experience is extraordinary**
+
+- js.ipfs.io is finished and published - @diasdavid
+- Onboard 2 new full time contributors - @diasdavid
+- Complete the Awesome Endeavour, Documentation
+- Cut CI testing time in js-ipfs and js-ipfs-api by 50% (2x)
```

Comment:
I'm all for this, but I think a precursor to this is **application/library code stability** and **test quality**, which would also contribute to the reliability objective.

In some cases, the **tests are not flakey**, the **application/library code is flakey** and the tests fail because of it.

In other cases the test assertions are false positives or false negatives or are not adequately testing responses or interactions.

Tests should fail when a code contribution causes an error, not when unrelated parts of our application/library code cause errors.

We're making good progress towards that already but the important thing to note here is that **we're not going to get a daemon running for 10 days without addressing the "flakey tests" problem**.

I'd like to put my name down for "A Daemon runs for 10 days without a crash" and "Cut CI testing time in js-ipfs and js-ipfs-api by 50% (2x)".

I feel these are super difficult to achieve given that they're dependent on a moving target (the codebase is constantly changing) but I'm willing to give them a shot.